types/hls.js/index.d.ts

```diff
@@ -354,12 +354,28 @@ declare namespace Hls {
     }
 
     interface AudioTrack {
```

Comment:
AudioTrack as mentioned in previous PR, was removed from TS and you don't need to leave it as is
Probably, the best solution here is to add `HlsAudioTrack`

Related PR: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/43335

Reply:
Applied requested changes. Is it ok now?

Reply:
LGTM